src/solc_0.8/common/BaseWithStorage/ERC2771/ERC2771HandlerUpgradeable.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+// solhint-disable-next-line compiler-version
+pragma solidity 0.8.2;
+
+/// @dev minimal ERC2771 handler to keep bytecode-size down
+/// based on: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.6.0/contracts/metatx/ERC2771Context.sol
+/// with an initializer for proxies and a mutable forwarder
+/// @dev same as ERC2771Handler.sol but with gap
+
+contract ERC2771HandlerUpgradeable {
+    address internal _trustedForwarder;
+    uint256[50] private __gap;
```

Comment:
shouldn't it be `uint256[49]` ?

Reply:
You are right, "The size of the __gap array is calculated so that the amount of storage used by a contract always adds up to the same number (in this case 50 storage slots)"